Sources/FluentSQL/SQLQueryConverter.swift

```diff
@@ -228,6 +228,16 @@ public struct SQLQueryConverter {
                     right: self.value(value)
                 )
             }
+        case .field(let lhsField, let method, let rhsField):
+            switch method {
+                // possible overrides for other methods
```

Comment:
Is anything specific needed here?

Reply:
i think we're fine to just return `SQLBinaryExpression` directly here.